transactional_licence_form/templates/nine_principles_1.html

```diff
@@ -44,10 +44,10 @@ <h3 id="dignity">1.Dignity of the court</h3>
       <li>Licence holders must not undermine the courts and tribunals’ ability to function as working bodies.</li>
     </ul>
     <p>
-      You should consider anything that may undermine the ability for the justice system to operate impartially and with integrity and treat all members of the public equally and fairly, no matter who they are
+      You should consider anything that may undermine the ability for the justice system to operate impartially and with integrity and treat all members of the public equally and fairly, no matter who they are.
     </p>
     <h3 id="independence">2. Independence of the court</h3>
-    <p>It is important that any analysis respects the independence of the judiciary and their impartial judgment.</p>
+    <p>It is important that any analysis respects the independence of the judiciary and their impartial judgement.</p>
```

Comment:
If this is a delibrate use of "judgement with an e" because it's not referring to a document, I guess it's fine